client/src/components/HandMarkedPaperBallot.tsx

```diff
@@ -33,6 +34,38 @@ import QRCode from './QRCode'
 import Prose from './Prose'
 import Text from './Text'
 
+const qrCodeTargetClassName = "qr-code-target"
+
+class PagedQRCodeInjector extends Handler {
+  afterRendered(pages: any) {
```

Comment:
(nice to have) Types?

Reply:
I want types. Whats the best way to do this? Can you give me some pointers here? PagedJS itself is not Typescript.

Reply:
Found the "infer from usage" option in VS Code to create types for the variables used. Thanks for the push.